llvm/test/CodeGen/AMDGPU/fma_combine.ll

```diff
@@ -0,0 +1,70 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
```

Comment:
Can you find a better place for this? We already have "fma-combine.ll" (with a hypen instead of an underscore). Maybe it could go in `fdot2.ll` which is where tests we added originally for this combine: https://reviews.llvm.org/D49146

Reply:
Thanks - I have updated fdot2.ll instead. With the added run line the test crashes without the fix.